app/src/services/auth-services.ts

```diff
@@ -6,7 +6,9 @@ const axios = AddAxiosInterceptors(Axios.create())
 
 export default class AuthServices {
   /** The Auth API URL. */
-  static authApiUrl = sessionStorage.getItem('AUTH_API_URL')
+  static get authApiUrl (): string {
+    return sessionStorage.getItem('AUTH_API_URL')
+  }
```

Comment:
I believe the old code assigned the property when this class is instantiated, which is before the session storage variable was populated.

Oddly, the old code works fine locally, so I need to deploy this to verify it.